### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by zacharyvoase** [thesayyn/protoc-gen-ts#145] [Q&A] [answered]
Would you be down to cut another release with the json_names feature? It'd be really helpful 😊 thank you so much!!

**Comment by thesayyn** [comment]
I'll cut a new release this week!

**Reply by zacharyvoase** [reply]
Just wanted to ask for an update on this, thanks :) although at the moment I'm getting by with a vendorized version of protoc-gen-ts

**Reply by thesayyn** [reply, marked as answer]
0.8.4 is on npm now! sorry, this slipped away.
